src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,70 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    /*  @FindBy annotation is used at the attribute level
+         to identify web elements in a page
+        @CacheLookup is used to cache the web element location results
+         (since the web page is static and its content will not change)
+     */
+    // web elements
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.